src/Uno.UI/Controls/UIFontHelper.iOS.cs

```diff
@@ -65,7 +65,7 @@ private static nfloat GetScaledFontSize(nfloat size, nfloat? basePreferredSize)
 			}
 
 			var originalScale = (basePreferredSize / DefaultUIFontPreferredBodyFontSize) ?? (float)1.0;
-			return (nfloat)(size * Math.Min(originalScale, FeatureConfiguration.Font.MaximumTextScaleFactor));
+			return size * Math.Min((float)originalScale, (float)FeatureConfiguration.Font.MaximumTextScaleFactor);
```

Comment:
add a non-null check for MaximumTextScaleFactor, same for src/Uno.UI/Extensions/ViewHelper.Android.cs

Reply:
or better yet, replace if (IgnoreTextScaleFactor) with if (MaximumTextScaleFactor is double/float factor)